src/scenes/ProgressReports/Detail/ProgressReportDetail.tsx

```diff
@@ -10,13 +10,12 @@ import {
   Tooltip,
   Typography,
 } from '@mui/material';
+// import { orderBy } from 'lodash';
 import { Helmet } from 'react-helmet-async';
 import { makeStyles } from 'tss-react/mui';
-import { Breadcrumb } from '../../../components/Breadcrumb';
-import {
-  idForUrl,
-  useChangesetAwareIdFromUrl,
-} from '../../../components/Changeset';
+import { ProgressReportBreadcrumb } from '~/components/progressReportBreadcrumb';
```

Comment:
Component folder names should be PascalCase.
```suggestion
import { ProgressReportBreadcrumb } from '~/components/ProgressReportBreadcrumb';
```